packages/commonwealth/test/integration/api/communityStake.spec.ts

```diff
@@ -58,43 +59,35 @@ describe('PUT communityStakes Tests', () => {
       expectedCreateResp.stake_enabled,
     );
 
-    const updateResp = await controller.putCommunityStake({
-      communityStake: { ...baseRequest, stake_token: 'temp' },
-      user: user,
-    });
-
-    const expectedUpdateResp = {
-      community_id: baseRequest.community_id,
-      stake_id: baseRequest.stake_id,
-      stake_token: 'temp',
-      vote_weight: baseRequest.vote_weight,
-      stake_enabled: baseRequest.stake_enabled,
-    };
-
-    assert.equal(updateResp.community_id, expectedUpdateResp.community_id);
-    assert.equal(updateResp.stake_id, expectedUpdateResp.stake_id);
-    assert.equal(updateResp.stake_token, expectedUpdateResp.stake_token);
-    assert.equal(updateResp.vote_weight, expectedUpdateResp.vote_weight);
-    assert.equal(updateResp.stake_enabled, expectedUpdateResp.stake_enabled);
-
-    const getResp = await controller.getCommunityStake({
+    try {
+      // try to change vote weight
+      await controller.createCommunityStake({
+        communityStake: { ...baseRequest, vote_weight: 20 },
+        user: user,
+      });
+    } catch (e) {
+      assert.equal(e.message, 'Community stake already exists');
+    }
```

Comment:
If it doesn't throw, then the catch statement will never be reached. The assertion should be performed outside of the try/catch statement.